server/config/express.middle.js

```diff
@@ -1,9 +1,13 @@
 'user strict'
 
 var bodyParser = require('body-parser');
+var express = require('express');
+var dirnameArray =  __dirname.split('/');
+var path = dirnameArray.slice(0, dirnameArray.length - 1).join('/');
```

Comment:
usually I see people using the node path module here: `var path = require('path')`
and then they do `path.join(__dirname, 'public')` down below